shared/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/hapi/HapiHelperTest.groovy

```diff
@@ -60,15 +60,15 @@ class HapiHelperTest extends Specification {
         actualMessageHeader == messageHeader
     }
 
-    def "getMSHMessageHeader throws a NoSuchElementException if the message header does not exist"() {
+    def "getMSHMessageHeader returns null if the message header does not exist"() {
```

Comment:
Since entire methods were moved out of `HapiHelper`, shouldn't entire tests be able to be deleted?

Reply:
Good catch. Cleaned up the tests